Sources/GRPC/ClientConnection.swift

```diff
@@ -27,9 +27,20 @@ import Logging
 /// The connection is initially setup with a handler to verify that TLS was established
 /// successfully (assuming TLS is being used).
 ///
-///                          ▲                       |
-///                HTTP2Frame│                       │HTTP2Frame
-///                        ┌─┴───────────────────────▼─┐
+///               ┌──────────────────────────┐
```

Comment:
Should we also add the source files for these diagrams to version control at some point?

Reply:
This is it I'm afraid! I created them with Monodraw originally and have just updated them manually since 😬

Pretty sure you can paste ASCII-art back into Monodraw though so it shouldn't be a problem.